Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/security-submissionadminreview.md

---
title: "submissionAdminReview resource type"
description: "Represents admin review information for threat submission"
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: resourcePageType
---

# submissionAdminReview resource type

Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents admin review information for threat submission. Currently, only user reported email threat submission is supported and could be reviewed by administrator.

Reviewer: ```suggestion
Represents admin review information for threat submission. Currently, only user reported email threat submission is supported and can be reviewed by administrator.
```

## Properties
| Property       | Type                     | Description                                  |
|:---------------|:-------------------------|:---------------------------------------------|

> | reviewBy       | String                   | The reviewer email or other identity string. |

Reviewer: ```suggestion
| reviewBy       | String                   | It specifies who reviewed the email? The identification is done using email id or other identity strings|
```

> | reviewDateTime | DateTimeOffset           | The review date time.                        |

Reviewer: ```suggestion
| reviewDateTime | DateTimeOffset           | It is used to specify the date time of when the review occurred.|
```

> | reviewResult   | submissionResultCategory | The review result                            |

Reviewer: ```suggestion
| reviewResult   | submissionResultCategory | It is used to specify what the review result was? |
```

> | reviewResult   | submissionResultCategory | The review result. The possible values are: `notJunk`, `spam`, `phishing`, `malware`, `allowedByPolicy`, `blockedByPolicy`, `spoof`, `unknown` and `noResultAvailable`.  |

Author: ```suggestion
| reviewResult   | submissionResultCategory | It is used to specify what the review result was? The possible values are: `notJunk`, `spam`, `phishing`, `malware`, `allowedByPolicy`, `blockedByPolicy`, `spoof`, `unknown` and `noResultAvailable`.  |
```
